FirebaseAdmin/FirebaseAdmin/AppCheck/AppCheckTokenVerify.cs

```diff
@@ -0,0 +1,263 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using FirebaseAdmin.Auth;
+using FirebaseAdmin.Auth.Jwt;
+using Google.Apis.Auth;
+using Google.Apis.Util;
+
+namespace FirebaseAdmin.Check
+{
+    internal class AppCheckTokenVerify
```

Comment:
```suggestion
    internal class AppCheckTokenVerifier
```

Reply:
I have updated file and class name, and the issue has been resolved.